projects/fluxcd/flux2/patches/0001-Replace-github.com-homeport-dyff-with-commit-to-work.patch

```diff
@@ -0,0 +1,39 @@
+From 86a5de2929b7b8b82f6e654c1aa306b57665548d Mon Sep 17 00:00:00 2001
+From: Jackson West <jgw@amazon.com>
+Date: Tue, 6 Jun 2023 12:23:43 -0500
+Subject: [PATCH] Replace github.com/homeport/dyff with commit to workaround go
+ mod signature failure
+
+---
+ go.mod | 2 ++
+ go.sum | 4 ++--
+ 2 files changed, 4 insertions(+), 2 deletions(-)
+
+diff --git a/go.mod b/go.mod
+index f65edc85..68eaff7e 100644
+--- a/go.mod
++++ b/go.mod
+@@ -217,3 +217,5 @@ require (
+ 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
+ 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
+ )
++
++replace github.com/homeport/dyff => github.com/homeport/dyff v1.5.7-0.20230415202303-35afb2435b21
```

Comment:
this newest version of dyff appears to have been republished which gets the go sum hashes out of date.  We hit this every now and then and the "fix" is to point to a commit either before or right after the release tag. This commit is the commit before the tag.